queries.sql

```diff
@@ -43,3 +43,96 @@ FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+SELECT * FROM animals; -- species column = unspecified
+ROLLBACK;
+
+--Inside a transaction: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon. Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+BEGIN;
+
+-- Update species to 'digimon' for animals with names ending in "mon"
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update species to 'pokemon' for animals with no species set
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
```

Comment:
- [ ]  In accordance to the project requirement, we are to verify if the changes persist before and after "`SET species = 'pokemon'`", to achieve this I think you can use the select query to check as suggested earlier. 🙏

- [ ] Kindly update the screenshot in the PR as stated in the requirement. Thanks 

<img width="715" alt="Untitled" src="https://github.com/Kingno3l/Vet-Clinic/assets/103386183/1314142a-416d-4656-8659-b25aa8a059ac">